arelle/CntlrCmdLine.py

```diff
@@ -57,16 +57,19 @@ def parseAndRun(args):
         hasWebServer = True
     except ImportError:
         hasWebServer = False
-    cntlr = CntlrCmdLine()  # need controller for plug ins to be loaded
-
+    uiLang = None
     # Check if there is UI language override to use the selected language
     # for help and error messages...
     for _i, _arg in enumerate(args):
-        if _arg.startswith('--uiLang'):
-            _uiLang = args[_i+1]
-            cntlr.setUiLanguage(_uiLang)
+        if _arg.startswith("--uiLang="):
```

Comment:
nit: probably a little safer to use a predefined variable for `'--uiLang'` and then compute all other usecases from that, but don't feel strongly